packages/feedback-service/package.json

```diff
@@ -57,7 +57,8 @@
     "method-override": "3.0.0",
     "mongoose": "5.9.16",
     "morgan": "1.10.0",
-    "nodemailer": "6.4.8",
+    "node-fetch": "^2.6.0",
+    "nodemailer": "6.4.6",
```

Comment:
Unused package. You can remove it.
Also make sure to run `npm install` after removing it so the package-lock.json is updated.